MiniProject/Modules/MenuDetail/DetailMenuViewController.swift

```diff
@@ -0,0 +1,194 @@
+            
+            contentView.topAnchor.constraint(equalTo: scrollView.topAnchor),
+            contentView.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor),
+            contentView.trailingAnchor.constraint(equalTo: scrollView.trailingAnchor),
+            contentView.widthAnchor.constraint(equalTo: scrollView.widthAnchor),
+            contentView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor),
+            
+            imageView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            imageView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            imageView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            imageView.heightAnchor.constraint(equalToConstant: 300),
+            
+            descriptionContainer.topAnchor.constraint(equalTo: imageView.bottomAnchor, constant: 16),
+            descriptionContainer.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            
+            descriptionLabel.topAnchor.constraint(equalTo: descriptionContainer.topAnchor, constant: 8),
+            descriptionLabel.leadingAnchor.constraint(equalTo: descriptionContainer.leadingAnchor, constant: 20),
+            descriptionLabel.trailingAnchor.constraint(equalTo: descriptionContainer.trailingAnchor, constant: -20),
+            descriptionLabel.bottomAnchor.constraint(equalTo: descriptionContainer.bottomAnchor, constant: -8),
+            
+            ingredientsHeading.topAnchor.constraint(equalTo: descriptionContainer.bottomAnchor, constant: 16),
+            ingredientsHeading.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            ingredientsHeading.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16),
+            
+            ingredientsContent.topAnchor.constraint(equalTo: ingredientsHeading.bottomAnchor, constant: 8),
+            ingredientsContent.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            ingredientsContent.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16),
+            
+            instructionsHeading.topAnchor.constraint(equalTo: ingredientsContent.bottomAnchor, constant: 16),
+            instructionsHeading.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            instructionsHeading.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16),
+            
+            instructionsContent.topAnchor.constraint(equalTo: instructionsHeading.bottomAnchor, constant: 8),
+            instructionsContent.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            instructionsContent.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -16),
+            instructionsContent.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -16),
```

Comment:
Wrong constraints, YouTube Button will be cropped with this code.
```suggestion
            youtubeButton.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -16),
```